### DISCUSSION TITLE
Lesson 4: couldn't find the AggregatorV3Interface on chainlink github

### DISCUSSION THREAD

**Discussion by ram-ali-khan** [Cyfrin/foundry-full-course-cu#1817] [Q&A]
i am unable to find the ''AggregatorV3Interface'' file on the chainlink repo on github as shown in the video.
https://github.com/smartcontractkit/chainlink/tree/develop/contracts/src/v0.8/interfaces

**Comment by urosognjenovic** [comment]
Here it is https://github.com/smartcontractkit/chainlink/tree/develop/contracts/src/v0.8/shared/interfaces.

**Comment by EngrPips** [comment]
Hello @ram-ali-khan, Do you find in the path @thebraveman17  helped with?
